sentry/src/db/event_aggregate.rs

```diff
@@ -16,26 +15,6 @@ use tokio_postgres::{
 
 use super::{DbPool, PoolError};
 
-pub async fn latest_approve_state(
-    pool: &DbPool,
-    channel: &ChannelOld,
-) -> Result<Option<MessageResponse<ApproveState>>, PoolError> {
-    let client = pool.get().await?;
-
-    let select = client.prepare("SELECT \"from\", msg, received FROM validator_messages WHERE channel_id = $1 AND \"from\" = $2 AND msg ->> 'type' = 'ApproveState' ORDER BY received DESC LIMIT 1").await?;
-    let rows = client
-        .query(
-            &select,
-            &[&channel.id, &channel.spec.validators.follower().id],
-        )
-        .await?;
-
-    rows.get(0)
-        .map(MessageResponse::<ApproveState>::try_from)
-        .transpose()
-        .map_err(PoolError::Backend)
-}
-
 pub async fn latest_approve_state_v5(
```

Comment:
Maybe we can rename `latest_approve_state_v5` and `latest_new_state_v5` to just `latest_approve_state` and `latest_new_state`?

Reply:
I would rather leave the `_v5` suffix for now otherwise more files will need to be changed and it will introduce merge conflicts